src/components/Dashboard/AssetStatusComponent.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { isEmpty } from 'lodash';
+import { Grid } from 'semantic-ui-react';
+
+import AnalyticsCardComponent from './AnalyticsCardComponent';
+
+export default class AssetStatusComponent extends Component {
+  componentDidMount() {
+    if (isEmpty(this.props.asset)) {
+      this.props.onFetchAssets(this.props.status);
+    }
+  }
+
+  render() {
+    const { asset, status } = this.props;
+    const activeClass = asset.count > 0 ? 'active-analytics' : '';
+
+    return (
+      <Grid.Column>
+        <div className={`analytics-state-rectangle ${activeClass}`}>
+          {asset && (
```

Comment:
Given that `asset` is an object and objects are always _truthy_, the checkmark will be shown even for status with a count of 0. I recommend that you assign `asset.count > 0` to a variable and use that. I.e.
```jsx
const hasAssets = asset.count > 0;
const activeClass = hasAssets ? 'active-analytics': '';

return (
  ....
  {hasAssets && (